app/models/Votes.scala

```diff
@@ -27,20 +27,68 @@ class Votes(tag: Tag) extends Table[Vote](tag, "Vote") {
     def foodPlace = foreignKey("FP_FK", placeId, Foodplaces.foodplaces)(_.id)
 }
 
+case class OpenChance(openChance: Double, numberOfVotes: Int) {
+    def toJson = Json.obj(
+        "openChance" -> openChance,
+        "numberOfVotes" -> numberOfVotes
+    )
+}
+
 object Votes {
 
     val votes = TableQuery[Votes]
 
     def getAll: List[Vote] = {
-      Database.forDataSource(DB.getDataSource()) withSession { implicit session =>
-          votes.list
-      }
+
+        Database.forDataSource(DB.getDataSource()) withSession { implicit session =>
+            votes.list
+        }
     }
 
     def getVotesFor(id: Int): List[Vote] = {
-      Database.forDataSource(DB.getDataSource()) withSession { implicit session =>
-          votes.list.filter(_.placeId == id)
-      }
+        Database.forDataSource(DB.getDataSource()) withSession { implicit session =>
+            votes.list.filter(_.placeId == id)
```

Comment:
Are these changes intended? If yes, consider adding codestyle idea's files to repo (I've excluded em recently)